content/pop-recovery.md

```diff
@@ -105,7 +106,7 @@ And now the existing hard drive can be accessed by going to the `/mnt` folder.
 
 ## Chroot
 
-`chroot` is the way to run commands as if the existing operating system had been booted.  Once these commands are run, then package manager (`apt`) and other system-level commands can be run.
+`chroot` runs commands as if the existing operating system had been booted.  Once these commands are run, then package manager (`apt`) and other system-level commands can be run.
```

Comment:
we don't need the "then" here.

Once these commands are run, package manager...